templates/jira/Bug.md

```diff
@@ -0,0 +1,23 @@
+# Bug ticket template
+
+Use this template for reporting a bug or regression. It is important to that the input describes what is observed, what is expected and how to reproduce the problem. Provide as much information as possible for a speedy resolution.
```

Comment:
Suggest: keep it factual (or something similar to keep emotions out of it)

Reply:
Added "observed objectively". See: https://github.com/pondevelopment/pon-developer-guide/pull/11/commits/ef14329caf5859cef42c096e41348600cc568590